### PR TITLE
Initial commit of helm charts for media-api-service

### PR DESCRIPTION
**Pull request by abes-slalom**
### Types of Changes / 변경 유형

- [x] Core / 핵심
- [ ] Bugfix / 버그 수정
- [ ] New component / 새 구성 요소
- [ ] Enhancement/optimization / 개선/최적화
- [ ] Documentation / 문서
- [ ] Test / 테스트

### Issues Addressed / 해결된 문제
* [VID-28|https://htw-cloud.atlassian.net/browse/VID-28]

### Checklist / 체크리스트

- [ x] Meets Design Standards / 설계 표준 충족
- [] Passes Tests / 테스트 통과
- [x ] Meets [Definition of Done](https://htw-cloud.atlassian.net/wiki/spaces/DEV/pages/3604642/Definition+of+Done) / [완료의 정의](https://htw-cloud.atlassian.net/wiki/spaces/DEV/pages/3604642/Definition+of+Done) 충족

### CODE REVIEW

File: charts/media-api-service/readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+A sample library for dealing with microfront ends that deploy behind traefik.
+
+## Installation
+
+clone the [helm-charts](https://github.com/htaic/helm-charts) repo into your source code and move the example file out:
+
+```bash
+git clone https://github.com/htaic/helm-charts.git
+mv ./helm-charts/charts/media-token-service/values.yaml ./DevValues.yaml
```
Comment (HanwhaARudolph) [inline charts/media-api-service/readme.md:11]: you should update the readme or remove it

File: charts/media-api-service/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.2
```
Comment (HanwhaARudolph) [inline charts/media-api-service/Chart.yaml:18]: should be 0.0.1 for the first publish

File: charts/media-api-service/templates/ingressroute.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{{ if and (ne .Values.image.domainPath "") (hasKey .Values.image "domainPath") }}
```
[the remark this answers is not in the document]
Comment (abes-slalom) [reply charts/media-api-service/templates/ingressroute.yaml:1]: We need it.

File: charts/media-api-service/readme.md (hunk 1 above)
Comment (abes-slalom) [reply charts/media-api-service/readme.md:11]: removed

File: charts/media-api-service/Chart.yaml (hunk 2 above)
Comment (abes-slalom) [reply charts/media-api-service/Chart.yaml:18]: updated

File: charts/media-api-service/values.yaml.Example [hunk 4]
```diff
@@ -0,0 +1,46 @@
+namespace: "hanwha"
+# fullnameOverride: imagename
+
+image:
+  replicaCount: 1
+  repository: ghcr.io/htaic/media-api-service
+  pullPolicy: Always
+  containerPort: 3000
+  tag: "latest"
+  domainPath: ""
```
Comment (HanwhaARudolph) [inline charts/media-api-service/values.yaml.Example:10]: Are you planning to expose this service or is it internal only... if its internal, remove domain path and the ingress route and middleware.

Also please confirm you are in fact using port 3000

File: charts/media-api-service/values.yaml.Example [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  livenessProbe: "api/v1/livez"
+  readinessProbe: "api/v1/readyz"
+  env:
```
Comment (HanwhaARudolph) [inline charts/media-api-service/values.yaml.Example:13]: is AWS_REGION actually a required arg?

File: charts/media-api-service/values.yaml.Example (hunk 4 above)
Comment (abes-slalom) [reply charts/media-api-service/values.yaml.Example]: Yep, that's NestJs' default port.

File: charts/media-api-service/values.yaml.Example (hunk 5 above)
Comment (abes-slalom) [reply charts/media-api-service/values.yaml.Example:13]: It is required.

File: charts/media-api-service/values.yaml.Example (hunk 4 above)
Comment (abes-slalom) [reply charts/media-api-service/values.yaml.Example]: The latest commit had the domainPath updated to "media-api".

**Pull request merged**